modules/oe_subscriptions_anonymous/tests/src/Kernel/TwigTemplatesTest.php

```diff
@@ -50,7 +63,8 @@ public function testLinkTemplate(): void {
       '',
       [
         'href' => $front_url->toString(),
-      ]
+      ],
+      $template_name
```

Comment:
Multi line argument list should have comma after the last item.
I thought this would be in automatic code style checks but seems it is not?